lib/github_service/commands/cross_repo_test.rb

```diff
@@ -270,25 +270,30 @@ def create_cross_repo_test_branch
       #
       #   https://github.com/ManageIQ/manageiq/blob/06de0607/lib/git_worktree.rb#L102-L110
       #
-      def update_travis_yaml_content
-        raw_yaml = rugged_repo.blob_at(branch_ref.target.oid, ".travis.yml").content
+      def update_github_workflow_yaml_content
+        raw_yaml = rugged_repo.blob_at(branch_ref.target.oid, ".github/workflows/ci.yml").content
```

Comment:
NOTE I'm assuming that this works as a directory path not a root file, need to confirm

Reply:
:+1: confirmed this works:
```
adam@desktop:~/src/manageiq/manageiq-cross_repo-tests-github-actions$ irb
>> require 'rugged'
=> true
>> rugged_repo = Rugged::Repository.new(".")
=> #<Rugged::Repository:240 {path: "/home/grare/adam/src/manageiq/manageiq-...
>> branch_ref = rugged_repo.branches["master"]
=> #<Rugged::Branch:260 {name: "master", target: #<Rugged::Commit:280 {mess...
>> rugged_repo.blob_at(branch_ref.target.oid, ".github/workflows/ci.yaml")
=> #<Rugged::Blob:0x000055962a376ef8 @owner=#<Rugged::Repository:240 {path: "/home/grare/adam/src/manageiq/manageiq-cross_repo-tests-github-actions/.git/"}>>
>> rugged_repo.blob_at(branch_ref.target.oid, ".github/workflows/ci.yaml").conte
nt
=> "name: Cross Repo Tests\n\non: [push, pull_request]\n\njobs:\n  manageiq:\n    uses: agrare/manageiq-cross_repo/.github/workflows/manageiq_cross_repo.yaml@github_actions\n    with:\n      test_repo: ManageIQ/manageiq@master\n      repos: ManageIQ/manageiq@master\n"
```